fastapi/app/__init__.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import sys
+
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger("spiceql")
+handler = logging.StreamHandler()
+formatter = logging.Formatter('SPICEQL [%(asctime)s] [%(levelname)s] [%(filename)s:%(lineno)d %(funcName)s] %(message)s', datefmt='%H:%M:%S %z')
```

Comment:
Maybe make the logs print `SPICEQL FASTAPI` so we know it's from the python side.